### DISCUSSION TITLE
How to get NNG Installed/Recognized on Windows?

### DISCUSSION THREAD

**Discussion by lordmichael95** [nanomsg/nng#1855] [Q&A]
Hello All,

I have a successfully installed NNG on a Linux system. I then developed a Python script which is correctly able to use NNG's Pair functionality (on this Linux system) in a git repository. However, when I clone in on a Windows system, this same script fails because it does not know about the NNG library (error message `import pynng ModuleNotFoundError: No module named 'pynng'`). This is even though the script is part of the same git repository. Therefore, it would seem that Windows systems needs some other steps to get the library installed. There are steps for getting the library on a Linux/Unix system https://github.com/nanomsg/nng, but not explicit steps for Windows. How do I get NNG installed/recognized on Windows?

**Comment by gdamore** [comment]
This is likely to be quite specific to your python installation.  I'm not familiar with the Python wrapper libraries unfortunately, including `pynng`.  You might try asking on the repository for `pynng`.
